src/istio/control/attribute_names.cc

```diff
@@ -28,6 +28,7 @@ const char AttributeName::kRequestPath[] = "request.path";
 const char AttributeName::kRequestReferer[] = "request.referer";
 const char AttributeName::kRequestScheme[] = "request.scheme";
 const char AttributeName::kRequestSize[] = "request.size";
+const char AttributeName::kReceivedBytes[] = "received.bytes";
```

Comment:
Hi @douglas-reid I felt it is better to name these two new attributes as "request.total_size" and "response.total_size" so it consists with existing attributes of "request.size"

Reply:
@qiwzhang `request.total_size` and `response.total_size` sgtm.